x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/management.js

```diff
@@ -86,15 +87,25 @@ export function actionsMenuContent(
         // the indexPattern the job was created for. An indexPattern could either have been deleted
         // since the the job was created or the current user doesn't have the required permissions to
         // access the indexPattern.
+        return item.deleting !== true && canCreateJob;
+      },
+      onClick: (item) => {
         const indexPatternNames = getIndexPatternNames();
         const jobIndicesAvailable = item.datafeedIndices.every((dfiName) => {
           return indexPatternNames.some((ipName) => ipName === dfiName);
         });
 
-        return item.deleting !== true && canCreateJob && jobIndicesAvailable;
-      },
-      onClick: (item) => {
-        cloneJob(item.id);
+        if (!jobIndicesAvailable) {
```

Comment:
This check doesn't seem to work when trying to clone a job which has been created from a comma-separated index pattern e.g. create an index pattern for `cloudwatch-2019.11.03, cloudwatch-2019.11.04, cloudwatch-2019.11.05`, then run a job against using that index pattern, and then try and clone it:
![image](https://user-images.githubusercontent.com/7405507/95101985-b9abd880-072a-11eb-8b5c-46456a52feb6.png)

Reply:
Fixed in 1eda34d90173330908c9fc0b5ab178a935fcbf53

Reply:
Fix works well for anomaly detection jobs created from a comma-separated index pattern, but it also also seems to affect the data frame analytics page. Any chance of applying the same fix there too?

Reply:
Good catch! 🙏  Updated! AD and DFA are now consistent. 1c7d2d94220fee04b27ba48baff0b7928e3df6de